docs/packages/development/ide_setup.md

```diff
@@ -0,0 +1,55 @@
+# IDE Setup for Python
+
+While a proper IDE is not strictly needed to develop python code,
+its usage will make the development process easier and more efficient, allowing the programmer to interact with the code in ways not possible with a text editor.
+
+
+## VSCode
+
+We recommend the usage of [VSCode][vscode_webpage]{target=_blank} for Python development, while other excellent IDEs exist,
+the following VSCode features are of great value:
+
+- [Remote SSH connection][vscode_remote_ssh]{target=_blank}:<br>
+  Use any machine with VSCode installed to connect to a remote machine and continue developing on it,
+  e.g. you can store your files on `afs` and connect to them through `lxplus`, working on them  as if they were present locally.
+  Check the [CERN Knowledge Base][vscode_lxplus]{target=_blank} for more information.
+- [Python support][vscode_python]{target=_blank}:<br>
+  Make use of static type checking and auto-completion. Interact with the code via an easy-to-use debugger.
+- [Interactive Python][vscode_interactive_python]{target=_blank}:<br>
+  Use any script as if it were a jupyter notebook, allowing you to run code snippets in real time and inspect variables.
+- [Integrated Testing][vscode_testing]{target=_blank}:<br>
+  Use the `pytest` framework to easily write and run tests, which can be individually run or all at once directly from the editor.
+- [Great Extendibility][vscode_marketplace]{target=_blank}:<br>
+  Many extensions are available, making VSCode a great tool for many different use cases and programming languages (e.g. [LaTeX][vscode_latex]{target=_blank}, [Java](../../guis/usage/ide_install.md#vscode),
+  which means you do not have to learn a new environment for your different projects.
+- Widespread use among the team:<br>
+  Most developers of the OMC-Team use VSCode and are happy to help you with setting up and using it.
```

Comment:
I would mention that an IDE gives you a debugger and that's a very cool plus

Reply:
added